### DISCUSSION TITLE
Website broken

### DISCUSSION THREAD

**Discussion by davidenco** [nolimits4web/swiper#5643] [General]
The link I have in my favourites (https://idangero.us/swiper/) has recently started loading a very old version of the website and advertising v4.5.1. Should it be?

**Comment by nolimits4web** [comment]
swiper website is at https://swiperjs.com, the one you mention is not active for years

**Reply by davidenco** [reply]
No worries, the old one was current when v8.1.0 was released, but I only noticed the different look today. I'll update my link.
